docs/components/LabelView.jsx

```diff
@@ -0,0 +1,199 @@
+import _ from "lodash";
```

Comment:
started adding them inline with the live examples (e.g. `<PropDocumentation>` below).
just added an introductory paragraph as well just as a brief explanation of how it can be used - https://github.com/Clever/components/pull/130/commits/3122c8896832d746c4981bc3acaed874bcef771a